### ISSUE TITLE
expected return code for redfish.account_service.test_user_account.Verify Error While Deleting Root User ?

### ISSUE THREAD

**Issue by generatz** [openbmc/openbmc-test-automation#2147]
This test is failing on 3 different platforms: two BMCs from different OEMs and romulus running on qemu.
The three are returning a 404 for the redfish `DELETE to /redfish/v1/AccountService/Accounts/root` but the test is expecting a 403.

Are the implementations correct or is the test correct?

**Comment by gkeishin** [comment]
@rahulmah can you check on this one

**Comment by gkeishin** [comment]
or @manashsarma @prkatti1 Have you seen this one in recent testing?

**Comment by manashsarma** [comment]
Yes, George . I think it is a similar issue I raised already through defect - 

SW530782 : FTC1020:Rainier: HTTP return code is not uniform in service related forbidden operations with Redfish

**Comment by rahulmah** [comment]
@manashsarma : That issue is specific to service login. We should have a separate one for root user to get this one fixed. 

@gkeishin : We should be getting response status code like 405 (Method Not Allowed) for this operation. At present we are getting not found error. Let me ask @AshwiniC777 to take care of this.

```                                                                                                                                                                                     rahulmaheshwari@Rahuls-MacBook-Pro ~ % curl -k -H "X-Auth-Token: $bmc_token" -X  DELETE https://${BMC_IP}/redfish/v1/AccountService/Accounts/root
{
  "error": {
    "@Message.ExtendedInfo": [
      {
        "@odata.type": "#Message.v1_1_1.Message",
        "Message": "The requested resource of type #ManagerAccount.v1_4_0.ManagerAccount named root was not found.",
        "MessageArgs": [
          "#ManagerAccount.v1_4_0.ManagerAccount",
          "root"
        ],
        "MessageId": "Base.1.8.1.ResourceNotFound",
        "MessageSeverity": "Critical",
        "Resolution": "Provide a valid resource identifier and resubmit the request."
      }
    ],
    "code": "Base.1.8.1.ResourceNotFound",
    "message": "The requested resource of type #ManagerAccount.v1_4_0.ManagerAccount named root was not found."
  }```

**Comment by justin0309** [comment]
@gkeishin 
![image](https://github.com/openbmc/openbmc-test-automation/assets/109942943/3269fcc9-5672-4a70-be3f-680d1fc147f0)
Is there any update? I also have the same error. Is it possible that the response code is wrong?

**Comment by gkeishin** [comment]
@rahulmah can you check on this and update

**Comment by justin0309** [comment]
The judgment of ec.value==5 should be added to this. Although I don’t know what 5 is, this is the result of the test. Another question is that the auto-test hopes to be 403. Does the current error message support 403?
https://github.com/openbmc/bmcweb/blob/20fa6a2c6cea77a4b411634dfca67eef6c8f430f/redfish-core/lib/account_service.hpp#L2102

**Comment by gkeishin** [comment]
Closing this.. PLease report new ticket if you still see it in latest . Thanks
